stenotype/packets.cc

```diff
@@ -238,6 +240,7 @@ Error PacketsV3::Builder::Bind(const std::string& iface, Packets** out) {
                                      &fanout_, sizeof(fanout_))),
                     "setting fanout");
   }
+  RETURN_IF_ERROR(SetPromisc(), "SetPromisc");
```

Comment:
If you wouldn't mind, I think it might be better to not do this in Bind(), and instead call SetPromisc, flag-controlled, from stenotype.cc.  I think the idea of setting promisc by default is probably best, but I'd like the ability to not do it.   Maybe add a --no_promisc flag, like the --no_watchdogs one?